Package.swift

```diff
@@ -637,73 +637,24 @@ let package = Package(
 
     .target(
       name: "FirebaseFirestoreTarget",
-      dependencies: [.target(name: "FirebaseFirestore",
-                             condition: .when(platforms: [.iOS, .tvOS, .macOS]))],
-      path: "SwiftPM-PlatformExclude/FirebaseFirestoreWrap"
-    ),
-
-    .target(
-      name: "FirebaseFirestore",
       dependencies: [
+        .target(
+          name: "FirebaseFirestore",
+          condition: .when(platforms: [.iOS, .tvOS, .macOS])
+        ),
+        .product(name: "abseil", package: "abseil-cpp-binary"),
+        .product(name: "gRPC-C++", package: "grpc-binary"),
+        .product(name: "nanopb", package: "nanopb"),
         "FirebaseCore",
         "leveldb",
-        .product(name: "nanopb", package: "nanopb"),
-        .product(name: "abseil", package: "abseil-cpp-SwiftPM"),
-        .product(name: "gRPC-cpp", package: "grpc-ios"),
       ],
-      path: "Firestore",
```

Comment:
What is the strategy for re-enabling source SPM builds locally?

Reply:
In this PR's current state, there isn't a good strategy as you'd have to effectively revert this PR. One thing I could do is take a similar approach to how we conditionally configure GoogleAppMeasurement and introduce an env var that will conditionally build Firestore from source rather than binary.

Also worth noting that if we build Firestore from source, the Abseil/gRPC xcframework headers don't work in their current state so we'd also conditonally need to load in the source dependencies. (Alternatively, we could do some header processing on the existing xcframework headers to make them usable from source, but that would require some tooling).

Reply:
Discussed offline. Decision is to hold off for now.